entrypoint.sh

```diff
@@ -133,19 +133,19 @@ if [[ "$1" != "bash" ]] && [[ "$1" != "sh" ]] ; then
     if [[ "$1" == "java" || "$1" == "/usr/bin/java" ]]; then
       shift # trim "java"
       if [[ "${PROFILER_ENABLED,,}" == "true" ]]; then
-        JAVA_OPTIONS="$JAVA_OPTIONS -javaagent:/app/diag/lib/qubership-profiler-agent.jar"
-        JAVA_OPTIONS="$JAVA_OPTIONS -Dprofiler.dump.home=/app/diag"
+        PROFILER_JVM_OPTIONS=("-javaagent:/app/diag/lib/qubership-profiler-agent.jar")
+        PROFILER_JVM_OPTIONS+=("-Dprofiler.dump.home=/app/diag")
       fi
-      echo "Run command:" java "$JAVA_OPTIONS" "$@"
-      java $JAVA_OPTIONS $@ &
+      echo "Run command:" java "${PROFILER_JVM_OPTIONS[@]}" "$@"
+      java "${PROFILER_JVM_OPTIONS[@]}" "$@" &
```

Comment:
I've renamed the option so it does not accidentally collide with options for the other docker image users